jupyterhub/README.md

```diff
@@ -90,3 +89,38 @@ Contains build chain manifest for CUDA enabled ubi 7 based images, provides `ten
 Contains build chain manifest for CUDA 11.0.3 enabled ubi 8 based images with python 3.8 support, provides `tensorflow-gpu` and `pytorch-gpu` enabled notebook image.
 
 *NOTE:* Builds in this overlay require 4-6 GB of memory
+
+## Deploying JupyterHub notebooks to custom namespaces
+
+JupyterHub has support for deploying notebooks to other namespaces. To help facilitate this, we provide a component under `jupyterhub/custom-notebook-deployment`. The steps to deploy are as follows:
+
+1. Create the namespace for JupyterHub to reside in.
+2. Create the namespace for the JupyterHub notebooks to reside in.
+3. Update the value of `notebook_destination` in `jupyterhub/jupyterhub/base/params.env` to be the same as the namespace created in step 2.
```

Comment:
I feel like after users do this step (and step 4), they will have difficulty creating a KfDef that points to their changes.  Isn't it possible to specify the parameter values in the KfDef?

Reply:
Updated. what do you think?